babel.config.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
-  presets: ["@babel/preset-typescript"],
+  presets: [
+    ["@babel/preset-env", { targets: { node: "14" } }],
```

Comment:
Is node 14 a suitable target here when we have defined v16 in the nvm file?

Reply:
Those are somewhat 2 different things:
- `.nvmrc` defines a node version that should be used when working in the repository, when working
- this target here defines what is the lowest common denominator for the node version that people can use when installing our packages

Reply:
Makes sense, thanks for the answer